express/properties/workflow.py

```diff
@@ -87,26 +84,31 @@ class PyMLTrainAndPredictWorkflow(WorkflowProperty):
         - pyml:post_processing:parity_plot:matplotlib: Creates a parity plot if the workflow is in "Training" mode.
     """
 
-    def __init__(self, name, parser, *args, **kwargs):
+    def __init__(self, name: str, parser: Any,
+                 *args,
+                 work_dir: str,
+                 upload_dir: str,
+                 object_storage_data: Dict[str, str],
+                 context_dir_relative_path: str,
+                 workflow: Dict[str, Any],
```

Comment:
same story here as above, these 5 properties are now excluded from being passed to the constructor, where before they were.